sdk/src/feature_set.rs

```diff
@@ -179,6 +183,7 @@ lazy_static! {
         (abort_on_all_cpi_failures::id(), "Abort on all CPI failures"),
         (use_loaded_executables::id(), "Use loaded executable accounts"),
         (turbine_retransmit_peers_patch::id(), "turbine retransmit peers patch #14631"),
+        (prevent_upgrade_and_invoke::id(), "Prevent upgrade and invoke in same tx batch"),
```

Comment:
maybe this?  nbd though
```suggestion
        (prevent_upgrade_and_invoke::id(), "require writeable program account during upgrade #14653"),
```